cfbs/build.py

```diff
@@ -1,5 +1,7 @@
 import os
+import logging as log
```

Comment:
*If input associated with the 'input' build step, no operation is done.* -- missing *no* in the commit message